frontend/src/app/api/share-page/og-image/route.tsx

```diff
@@ -0,0 +1,34 @@
+import { NextResponse } from 'next/server';
```

Comment:
Explanation: The route makes a call to the custom OG image template(ID: 10) in Orshot to generate and get back image binary